spec/system/signed_in_user_creates_a_recipe_spec.rb

```diff
@@ -1,21 +1,39 @@
 # frozen_string_literal: true
 
-require "rails_helper"
-require "support/features/clearance_helpers"
+require 'rails_helper'
+require 'support/features/clearance_helpers'
 
-RSpec.feature "A signed in user creates a recipe" do
-  scenario "with valid parameters" do
+RSpec.feature 'A signed in user creates a recipe' do
+  scenario 'with valid parameters, but no picture' do
     user = create(:user)
     recipe = FactoryBot.build(:recipe)
 
     sign_in_with user.email, user.password
-    visit "/"
-    click_link I18n.t("layouts.application.new_recipe")
-    fill_in "Title", with: recipe.title
-    fill_in "Ingredients", with: recipe.ingredients
-    fill_in "Instructions", with: recipe.instructions
-    click_button "Create"
+    visit '/'
+    click_link I18n.t('layouts.application.new_recipe')
+    fill_in 'Title', with: recipe.title
+    fill_in 'Ingredients', with: recipe.ingredients
+    fill_in 'Instructions', with: recipe.instructions
+    click_button 'Create'
 
-    expect(page).to have_current_path("/")
+    expect(page).to have_current_path('/')
+  end
+
+  scenario 'and includes a picture' do
+    user = create(:user)
+    recipe = FactoryBot.build(:recipe)
```

Comment:
`FactoryBot.attributes_for` is maybe 0.00000000001 times faster than `FactoryBot.build`, in case you want that.